src/COCOA/Controllers/LoginController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+
+namespace COCOA.Controllers
+{
+    //[RequireHttps]
+    //[Authorize(Policy = "Authenticated")]
+    public class LoginController : Controller
```

Comment:
Consider moving this to UserController and add a new Login-view from it in the future.